src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const basicDayCost = 40;
+  const fullCost = days * basicDayCost;
+  const regularDiscount = 20;
+  const weekDiscount = 50;
+
+  const regularMinDays = 3;
+  const weekMinDays = 7;
```

Comment:
What if we change the number of discount days? (for example, for 30days-month)

You should use abstract variable naming